app/javascript/packs/micropost.js

```diff
@@ -0,0 +1,6 @@
+$('#micropost_image').bind('change', function() {
+  var size_in_megabytes = this.files[0].size/1024/1024;
+  if (size_in_megabytes > 5) {
+    alert(i18n.t('alert_image_size'));
```

Comment:
```suggestion
    alert(I18n.t('alert_image_size'));
```
không check lại (tat)